enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/helper/ErrorHandler.java

```diff
@@ -68,4 +83,9 @@ private void throwIfException()
             throw runtimeException;
         }
     }
+
+    public interface ThrowingRunnable
```

Comment:
You already import ThrowingAction, which perhaps you intended to use?

Reply:
Throwing action extends exception and not throwable unfortunately